homeassistant/components/cast/__init__.py

```diff
@@ -88,7 +88,7 @@ async def async_play_media(
         hass: HomeAssistant,
         cast_entity_id: str,
         chromecast: Chromecast,
-        media_type: str,
+        media_type: MediaType | str,
```

Comment:
I think it would make sense to adjust the `media_type` annotation here as well then.
https://github.com/home-assistant/core/blob/9a5fe950a4895b3bc20664f9d41605998049a1fe/homeassistant/components/cast/media_player.py#L636-L638